i18n/config.py

```diff
@@ -59,7 +60,7 @@ def read_config(self, filename):
         """
         if not os.path.exists(filename):
             raise Exception("Configuration file cannot be found: %s" % filename)
-        with open(filename) as stream:
+        with io.open(filename, encoding='UTF-8') as stream:
```

Comment:
Just curious, but why can we no longer use the built-in `open` method?

Reply:
@amangano-edx  In built-in open encoding argument was introduced in 3 onward versions. so it fails for earlier versions python 2. To make it generic used io.open which has encoding argument for both python2 and python3